#include <lwr_controllers/RPY.h>

string base_name
string tool_name
geometry_msgs/Vector3 position
RPY orientation

